### ISSUE TITLE
Necessary to clear browser history for updated catalog to display correctly

### ISSUE THREAD

**Issue by tmheunis** [abcd-j/data-catalog#19]
After the recent update to the abcd-j catalog, when I navigated to the main page (data.abcd-j.de) and the old version was still showing up. A refresh didn't work, I had to clear browsing data and then refresh the page and then the new homepage version showed up. I wonder if users will also have this issue and how we should deal with it. Is there a way to deal with this programmatically? @mslw did you encounter this with the SFB1451 catalog?

**Comment by mslw** [comment]
Yes, I am seeing it all the time when I'm checking updates for the SFB1451 catalog locally before pushing them to the published version. Thanks for bringing it up!

For me the easiest way to get an updated version is to close and open the browser. So, given that the cache seems to be limited to a browser session (?) I'm assuming users will be less likely to see this effect.

I am mostly using Firefox, and I started using private Chrome windows for doing such previews with maximum separation, so that I don't have to close all other tabs which I currently have open, and don't have old versioned URLs suggested to me. Firefox developer tools (F12) have an option inside their settings to "Disable HTTP Cache (when this toolbox is open)", which also worked for me when I was looking into some of the page's details. But typing that in made me realize that these are just workarounds for something I never asked about because I got used to doing them...

**Comment by jsheunis** [comment]
@aqw I'll be interested to hear if you have opinions about this

**Comment by aqw** [comment]
`no-cache` sounds sane (`reload` in worst case).

This can be done server-side too, but IMO it's a better fit client side.

**Comment by jsheunis** [comment]
This has been addressed in `datalad-catalog/main` by https://github.com/datalad/datalad-catalog/pull/458. This issue can be closed once those commits are applied here as well.
